### DISCUSSION TITLE
When updating a resources for second+ time, clients with system time set few minutes in the past get the error: "Another user has updated this resource since this page was loaded. Please refresh the page and try again."

### DISCUSSION THREAD

**Discussion by geethubert292** [laravel/nova-issues#3204] [Q&A] [answered]
- Laravel Version: 8.25.0
- Nova Version: 3.20.0
- PHP Version: 7.4.9
- Browser type and version: Chrome, Firefox

### Detailed steps to reproduce the issue on a fresh Nova installation:
Important: wouldn't work if server is on same computer with client, as the time is same
1. Set system time 5 minutes in the past
2. Update same resource several times in the row
3. You basically can update same resource once each 5 minutes?!

Having the system time set in the past doesn't seem to have any practical use, but there are users like that for some reason, and the text of the error doesn't help figuring out the cause of the problem for them.

**Comment by crynobone** [comment, marked as answer]
Laravel Nova implements Traffic Cop feature to prevents a user from modifying a resource that has been updated by another user, the `updated_at` field is used to identify if the resource has been modified. 

If you have to use Nova with the above setup please disable Traffic Cop. See https://nova.laravel.com/docs/3.0/resources/#disabling-traffic-cop
